app.js

```diff
@@ -0,0 +1,28 @@
+import getRandomThrow from '/.get-random-throw.js';
```

Comment:
need to switch dot and slash to have proper pathing --> './get-random-throw.js';